__tests__/integration.js

```diff
@@ -103,6 +103,50 @@ describe.each(EACH_MATRIX)('%s:%s: integration tests', (eventSourceName, framewo
     expect(response).toEqual(expectedResponse)
   })
 
+  test('GET Uint8Array', async () => {
+    const multiValueQueryStringParameters = {
+      singleNormal: ['1']
+    }
+    const queryStringParameters = {
+      singleNormal: '1'
+    }
+    router.get('/uint8', (req, res) => {
+      const { singleNormal } = req.query
+      res.set('X-Custom-Header', singleNormal)
+
+      const readStream = new stream.PassThrough()
+      const buffer = Uint8Array.from(
```

Comment:
https://github.com/nodejs/node/issues/14118#issuecomment-313933800 might help.

Reply:
I've seen this behavior of `buffer instanceof Uint8Array` covering cases of any buffer happening in my library too, Webstorm also tells me to remove `data instanceof Uint8Array` and leave just `Buffer.isBuffer(data)` but my tests failed when I passed `Uint8Array`, so you need to check this instanceof.

Well, maybe it's the internal behavior of express that makes this new validation not marked as tested, should we leave it as is or try to mark it in the test?

Reply:
Yeah, it looks like this issue comes from special behaviors of express and Next.js.
The original issue I posted mentioned Next.js (and the issue also happens in our Next.js apps), so if we wanted to have this particular case covered by tests, we would need to somehow replace express by Next.js (or some other app), which does not convert Uint8Array to Buffer.

So the test you added makes sense if the `if (Buffer.isBuffer(data)` case has not been tested elsewhere.
If the added test is only for covering ` || data instanceof Uint8Array`, I personally would remove it

Reply:
When I reverse the validation to check the Uint8Array first, the validation is marked as tested:

![image](https://user-images.githubusercontent.com/12551007/180452515-d76bd413-a7a3-40a3-8650-334c8509017e.png)

But if I remove my test, the validation will still be marked as tested, so I'll remove my test because it's not actually testing anything.

Let's wait for Brett to see if he has any ideas on how to properly test this line.